Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private string _name;
+        public string GetName()
+        {
+            return _name;
+        }
+        public void SetName(string name)
+        {
+            _name = name;
+        }
+
+        private decimal[] _balances;
+        public decimal[] GetBalances()
+        {
+            return _balances;
+        }
+        public void SetBalances(decimal[] balances)
+        {
+            _balances = balances;
+        }
+
+        public decimal GetCurrentBalance()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetCurrentBalance' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)

Reply:
I think we are supposed to be using Get and Set methods.  I could have sworn the instructor said to use that because I remember somebody asking about using property instead and I was thinking somebody was talking about real estate.